clients/src/main/java/org/apache/kafka/common/metrics/stats/Percentiles.java

```diff
@@ -39,10 +39,6 @@
     private final Percentile[] percentiles;
     private final BinScheme binScheme;
 
-    public Percentiles(int sizeInBytes, double max, BucketSizing bucketing, Percentile... percentiles) {
```

Comment:
Ditto on this constructor being public.
We recently discussed leaning on the safe side and not risking breaking source compatibility in the KIP-488 discussion thread here http://mail-archives.apache.org/mod_mbox/kafka-dev/201907.mbox/%3CCAAyirGtvLk-syqcC3g_biCjxR%3DhRgBOBkY_DQssWLvnsG5KgZQ%40mail.gmail.com%3E